config/configuration.json

```diff
@@ -925,9 +925,19 @@
       "mediumPriorityRule": {
         "condition":"OR",
         "rules":["highway=primary","highway=primary_link","highway=secondary","highway=secondary_link"]
-      },
+      }},
       "tags":"highway"
-    }
+  },
+  "SuddenHighwayTypeChangeCheck":{
+    "enabled": true,
```

Comment:
This is not necessary, as it will be enabled by default.

Reply:
updated!